showcase/src/app/demo-ui/pages/collapsible/demo-collapsible-page.component.html

```diff
@@ -2,7 +2,7 @@ <h1 class="mat-display-3" translate>SHOWCASE.DEMO.COLLAPSIBLE.TITLE</h1>
 <section class="stark-section">
 	<h1 translate>SHOWCASE.DEMO.SHARED.EXAMPLE_VIEWER_LIST</h1>
 	<example-viewer
-		[extensions]="['HTML', 'CSS', 'TS']"
+		[extensions]="['HTML', 'SCSS', 'TS']"
```

Comment:
In fact, looking at the SCSS code, I think we can simply remove this file: `showcase/src/assets/examples/collapsible/collapsible-default.scss` because those styles are not relevant for the example